config/production.yaml.example

```diff
@@ -639,6 +639,10 @@ client:
       # By default PeerTube client displays author username
       prefer_author_display_name: false
       display_author_avatar: false
+    upload:
+      # Max size, given in bytes, of upload chunks
+      # If not set it will be calculated based on network speed
+      max_chunk_size: 0
```

Comment:
* Prefer using `null` value instead of `0`
 * I think we should parse bytes here
 * Add an example: (`'100MB'` for example)

Reply:
In the quota config we're using `-1` to "unset" the value, may it be inconsistent to use null here?

Reply:
`-1` for quota means "Unlimited", not "unset" :)